### PR TITLE
Fix cross thread exception on update of hubhop presets

### LINKED ISSUE #1500: Cross thread exception on progress after starting update huphob presets older than 7 days
**Describe the bug**
Cross thread exception on progress after starting update huphob presets older than 7 days

**Steps To Reproduce**
Might happen after startup of mobiflight when the reminder to update the older then 7 days huphob presets appears. Probably might also happen when starting the update from the menu.

**Expected behavior**
Progress should be shown and presets updated.

**Actual behavior**
Progress does not show, exception does happen.

**Details**
 - MobiFlight Version: some dev version...

**Additional context**
image

### PR DESCRIPTION
**Pull request by Koseng**
fixes #1500

### CODE REVIEW

File: UI/Forms/ProgressForm.cs [hunk 1]
```diff
@@ -46,8 +42,7 @@ public void OnProgressUpdated(object sender, ProgressUpdateEvent e)
 
             if (e.Current == e.Total)
             {
-                if (this.InvokeRequired)
-                    this.Invoke(new EventHandler<ProgressUpdateEvent>(OnProgressCompleted), new object[] { sender, e });
+                this.Invoke(new EventHandler<ProgressUpdateEvent>(OnProgressCompleted), new object[] { sender, e });
```
Comment (DocMoebiuz) [inline UI/Forms/ProgressForm.cs:45]: if this cannot be called from a different thread context (we checked above) do:

`OnProgressCompleted?.Invoke(sender, e)`

**Review by DocMoebiuz — CHANGES_REQUESTED** [review]
see my comment

File: UI/Forms/ProgressForm.cs (hunk 1 above)
Comment (Koseng) [reply UI/Forms/ProgressForm.cs:45]: It's even simpler:
`OnProgressCompleted(sender, e);`

**Review by DocMoebiuz — CHANGES_REQUESTED** [review]
Please use the `?.Invoke` style otherwise this will throw an exception in case no handler is registered

File: UI/Forms/ProgressForm.cs (hunk 1 above)
Comment (DocMoebiuz) [reply UI/Forms/ProgressForm.cs]: That's not a good practice of Invoking an EventHandler - it works when you're absolutely 100% sure that a handler was registered before. I didn't check if this is the case, but let's stick to common practice.

Comment (Koseng) [reply UI/Forms/ProgressForm.cs]: There is no Event which can be called in ProgressForm. There only is the method.

**Review by DocMoebiuz — APPROVED** [review]
Thanks!

**Pull request merged**